mariadb/content.md

```diff
@@ -8,42 +8,53 @@ The intent is also to maintain high compatibility with MySQL, ensuring a library
 
 # How to use this image
 
-The %%IMAGE%% has a number of tags, and of note is `latest`, as the latest stable version, and `lts`, as the last long term support release.
+> [!IMPORTANT]
+> The %%IMAGE%% has a number of tags, and of note is `latest`, as the latest stable version, and `lts`, as the last long term support release.
```

Comment:
The target for this description is Docker Hub, not GitHub, so this GitHub-specific syntax probably won't land well.

Reply:
Does 'last' -> 'latest' make more sense? I'm hoping there are going to be more LTS releases :-)

Reply:
@tianon Ok, I'll remove it

@grooverdan it was written last, I just changed the formatting to put IMPORTANT. If you prefer, I can change like you wrote.

Reply:
I'd appreciate it if you did. Thanks, it was probably my wording mistake.